sdk/python/setup.py

```diff
@@ -41,6 +41,20 @@
       'kfp_run.api',
       'kfp_run.models',
     ],
+    classifiers=[
+      'Intended Audience :: Developers',
+      'Intended Audience :: Education',
+      'Intended Audience :: Science/Research',
+      'License :: OSI Approved :: Apache Software License',
+      'Programming Language :: Python :: 3',
+      'Programming Language :: Python :: 3.6',
```

Comment:
I think the SDK is good on py 3.5 too. Add 3.5?

Reply:
Done